zombienet_tests/functional/0004-slashing.toml

```
@@ -0,0 +1,46 @@
+[settings]
+timeout = 1000
```

I think we can just add more time here to allow the k8s cluster to scale out.
```suggestion
timeout = 1600
```